pyalgotrading/algobulls/connection.py

```diff
@@ -271,7 +279,94 @@ def get_report(self, strategy_code, trading_type, report_type, render_as_datafra
             print('Report not available yet. Please retry in sometime')
             # return response
 
-    def backtest(self, strategy=None, start=None, end=None, instruments=None, lots=1, parameters=None, candle=None, mode=StrategyMode.INTRADAY, delete_previous_trades=True, **kwargs):
+    def get_pnl_report_table(self, strategy_code, trading_type):
+        """
+            Fetch Back Testing Profit & Loss details
```

Comment:
"Back Testing" not right as this function is used by BT/PT/RT all

Reply:
done